src/config.rs

```diff
@@ -15,12 +15,63 @@ use trussed::{
     Client,
 };
 
+#[derive(Debug)]
+/// Structure meant to be stored in  a `static` to signal applications that they have been factory-resetted by the admin app
+pub struct ResetSignalAllocation(AtomicU8);
```

Comment:
AFAIS there is no way to construct this currently.  Implement `Default` and/or add `new`?